ddev/tests/cli/validate/test_licenses.py

```diff
@@ -66,9 +66,7 @@ def test_validate_repo(repo, expected_message, ddev, helpers, config_file):
 
 
 def test_error_no_requirements_file(repository, ddev, helpers):
-    agent_requirements_path = (
-        repository.path / 'datadog_checks_base' / 'datadog_checks' / 'base' / 'data' / 'agent_requirements.in'
-    )
+    agent_requirements_path = repository.path / 'agent_requirements.in'
```

Comment:
The changes in this test file cause test failures which are pretty much unavoidable by way of how the `repository` fixture works: it checks out `master`, which obviously doesn't have the necessary change, a catch-22 as there's no way to change this without major changes to the test or the fixture.

Reply:
I'll start with these tests when dealing with the repo clone fixture this week.